sdk/python/v1beta1/examples/bayesianoptimization-katib-sdk.ipynb

```diff
@@ -0,0 +1,531 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "from kubernetes.client import V1ObjectMeta\n",
+    "import kubeflow.katib as kc\n",
+    "from kubeflow.katib import constants\n",
+    "from kubeflow.katib import utils\n",
+    "from kubeflow.katib import V1alpha3AlgorithmSetting\n",
```

Comment:
Do you need to change it to v1beta1?

Reply:
I've removed v1beta1 examples temporarily because I met some obstacles on setup environments for these notebook examples.

Reply:
Ok.
@sperlingxx can you create as issue to add examples for v1beta1 SDK, please?

Reply:
@andreyvelich It's Done https://github.com/kubeflow/katib/issues/1300 .